tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -26,10 +26,20 @@ def _check_cpuid_x86(test_microvm, expected_cpu_count, expected_htt):
 
 
 def _check_cpu_features_arm(test_microvm):
-    expected_cpu_features = {
-        "Flags": "fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp "
-        "asimdhp cpuid asimdrdm lrcpc dcpop asimddp ssbs",
-    }
+    _, stdout, stderr = utils.run_cmd("lscpu")
+    assert stderr == ""
+
+    if "sve" in stdout:
```

Comment:
I d rather use the instance type to compute expected guest cpu features rather than the presence of sve. It is more future proof IMO.

Reply:
You are right! Done.